sys/bloom/bloom.c

```diff
@@ -35,7 +35,7 @@ bloom_t *bloom_new(size_t size, size_t num_hashes, ...)
     }
 
     /* Allocate Bloom array */
-    if (!(bloom->a = calloc(ROUND(size), sizeof(char)))) {
+    if (!(bloom->a = calloc(sizeof(uint8_t), ROUND(size)))) {
```

Comment:
I did not look at the other parts of this PR, and for the bloom filter change: I have no strong opinion on the diff. I am OK with leaving this like @Kijewski specified (assuming the tests do not fail)